### ISSUE TITLE
Flags should be `kebab-case` instead of `camelCase`

### ISSUE THREAD

**Issue by elijaharita** [filecoin-project/motion#193]
Camel case flags are definitely not a standard or expected practice. Will it break anything to change this? We should do it sooner than later.

**Comment by elijaharita** [comment]
On this topic, some Singularity-related flags are prefixed as `experimentalSingularityXXX`. Are these prefixes still indicating something useful, or can we remove the `experimental` part?

**Comment by rvagg** [comment]
yes please, I was just in there and there's even a mix of casings for flags
